### ISSUE TITLE
Missing library for matplotlib

### ISSUE THREAD

**Issue by frheault** [scilus/scilpy#489]
On brand new computer (Ubuntu 20.04 and up) I saw that `libfreetype6-dev` was missing and was leading to an error for installing matplotlib.

Did anyone else have that error? Should it be added after openblas & al. in the README.md?

It gets solve with `sudo apt-get install libfreetype6-dev`

**Comment by CHrlS98** [comment]
Yes, I've also had that issue on machines without a desktop environment (containers more precisely). However, I do not think I get the error when installing `scilpy` following a fresh *normal* Ubuntu installation (as opposed to *minimal*).

**Comment by jchoude** [comment]
I do get the same error with a fresh Ubuntu normal install.
